alephium/contracts/token_bridge/token_bridge_governance.ral

```diff
@@ -115,4 +116,23 @@ Abstract Contract TokenBridgeGovernance(
         let consistencyLevel = u256From1Byte!(byteVecSlice!(payload, 33, 34))
         minimalConsistencyLevel = consistencyLevel
     }
+
+    pub fn updateRefundAddress(vaa: ByteVec) -> () {
+        let (targetChainId, payload) = parseAndVerifyGovernanceVAA(vaa, ActionId.UpdateRefundAddress)
+        assert!(targetChainId == localChainId, ErrorCodes.InvalidTargetChainId)
+        let addressBytes = byteVecSlice!(payload, 33, 66)
+        // only support P2PKH address
+        assert!(byteVecSlice!(addressBytes, 0, 1) == #00, ErrorCodes.InvalidUpdateRefundAddressMessage)
+        let newRefundAddress = byteVecToAddress!(addressBytes)
+        let chainSize = u256From1Byte!(byteVecSlice!(payload, 66, 67))
+        assert!(size!(payload) == 67 + chainSize * 2, ErrorCodes.InvalidMessageSize)
+
+        let mut offset = 67
+        for (let mut i = 0; i < chainSize; i = i + 1) {
+            let chainIdBytes = byteVecSlice!(payload, offset, offset + 2)
+            let tokenBridgeForChainId = subContractId!(Path.TokenBridgeForChain ++ chainIdBytes)
+            TokenBridgeForChain(tokenBridgeForChainId).updateRefundAddress(newRefundAddress)
```

Comment:
Is it better to put `refundAddress` in `TokenBridge.ral` so that we only need to update it once?

In most of the cases, token bridge is loaded, so it's not very costly to get refundAddress from `TokenBridge`.

Reply:
Yes, it's better to put `refundAddress` in `TokenBridge` contract, thanks!